core/src/main/scala/org/apache/spark/scheduler/TaskSetManager.scala

```diff
@@ -483,8 +483,9 @@ private[spark] class TaskSetManager(
           // a good proxy to task serialization time.
           // val timeTaken = clock.getTime() - startTime
           val taskName = s"task ${info.id} in stage ${taskSet.id}"
-          logInfo("Starting %s (TID %d, %s, %s, %d bytes)".format(
-              taskName, taskId, host, taskLocality, serializedTask.limit))
+          logInfo("Starting %s (TID %d, %s, %d, %s, %d bytes)".format(
```

Comment:
No, I mean why how use the same `s"..."` syntax as in the line above? `Int` is fine.